pkg/systemd/logs.scss

```diff
@@ -56,6 +56,9 @@
 
     .pf-c-description-list {
         --pf-c-description-list--m-horizontal__term--width: 25ch;
+        border-top: 1px solid var(--pf-global--BorderColor--100);
```

Comment:
This adds a weird border to the description lists when it's under tabs. 
![Screen Shot 2021-11-18 at 16 27 34](https://user-images.githubusercontent.com/14921356/142445058-3572da58-33a6-4439-aa0d-27b68d32c267.png)

Reply:
I suggest let's revert this commit, as it's actualy changing the design from before, but also breaking the abrt logs details.

Reply:
I was afraid of that. I couldn't find some entry to trigger those tabs.

Could we make it conditional to not having tabs? I guess add a `:first-child` to it, perhaps, if the tabs are at the same level? Or have it be a class the JSX adds? The styling of the border and margin/padding should be in the same style.

Reply:
It's fine if we even drop the commit to merge this in. But I'd like a little round of polishing up afterward if that's the case. (But it is simple enough that we might be able to include it with a minor modification in this PR?)

Reply:
I actually think that you don't need border in the list but a horizontal ruler with spacers, and that only in that simple cards, (not abrt).